packages/trackerless-network/src/logic/neighbor-discovery/Handshaker.ts

```diff
@@ -94,17 +98,18 @@ export class Handshaker implements IHandshaker {
         return targetNeighbors.map((neighbor) => this.createRpcRemote(neighbor.getPeerDescriptor()))
     }
 
-    private async doParallelHandshakes(targets: HandshakeRpcRemote[], excludedIds: NodeID[]): Promise<NodeID[]> {
-        const results = await Promise.allSettled(
-            Array.from(targets.values()).map(async (target: HandshakeRpcRemote, i) => {
-                const otherNode = i === 0 ? targets[1] : targets[0]
-                const otherNodeId = otherNode ? getNodeIdFromPeerDescriptor(otherNode.getPeerDescriptor()) : undefined
-                return this.handshakeWithTarget(target, otherNodeId)
-            })
-        )
+    private async doParallelHandshakes(targetOne: HandshakeRpcRemote, targetTwo: HandshakeRpcRemote, excludedIds: NodeID[]): Promise<NodeID[]> {
```

Comment:
Could we get rid of the mutation of `excludedIds` parameter? What this and other `Promise<NodeID[]>` methos actually return?